contracts/eosio.system/src/producer_pay.cpp

```diff
@@ -88,6 +88,8 @@ namespace eosiosystem {
          auto to_voters        = 2 * to_per_block_pay;
          auto to_savings       = distribute_tokens - (to_voters + to_per_block_pay);
 
+         auto to_producers_pay = to_per_block_pay * 21 * STANDBY_PAY_RATIO_DENOMINATOR / ( _gstate4.standby_pay_ratio_numerator * _gstate4.num_standby_slots + 21 * STANDBY_PAY_RATIO_DENOMINATOR);
```

Comment:
Let's add a comment above this line explaining the forumla:
// to_producers_pay = [1% inflation] / (21 + num_standby_slots * standby_pay_ratio);
// where standby_pay_ratio = STANDBY_PAY_RATIO_DENOMINATOR / standby_pay_ratio_numerator

Reply:
yeah ok let's revert to @quocle108 's suggestion. It is confiusing